src/components/store/pages/callbacks/success.js

```diff
@@ -1,26 +1,54 @@
 import React, { useState, useEffect } from "react";
 import { useLocation } from "react-router-dom";
+import { faSpinner } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import Page from "../../page";
 
 export default () => {
   const query = new URLSearchParams(useLocation().search);
   const creditKeyId = query.get("id");
+  const [loading, setLoading] = useState(false);
   const [checkoutStatus, setCheckoutStatus] = useState(undefined);
 
   useEffect(() => {
     // complete order with backend
-    fetch(`https://c2rxvcaph7.execute-api.us-west-2.amazonaws.com/staging/demo-success?ckkey=${creditKeyId}`)
-      .then((res) => res.json())
-      .then((json) => {
-        setCheckoutStatus(json);
-      });
+    if (creditKeyId) {
+      setLoading(true);
+      fetch(`${process.env.REACT_APP_BACKEND_URL}${creditKeyId}`)
+        .then((res) => {
+          setCheckoutStatus(res);
```

Comment:
does this need to call `res.json()` so that it gets parsed to a JS object first?